cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -184,13 +167,20 @@ def _remove_docker_creds():
 
 
 @pytest.fixture(scope="module")
-def local_setup_provider(request, setup_provider_modscope, provider, appliance):
+def local_setup_provider(request, ssa_single_vm, setup_provider_modscope, provider, appliance):
 
-    # TODO: allow for vddk parameterization
     if provider.one_of(VMwareProvider):
+        view = navigate_to(ssa_single_vm, 'Details')
+        host_name = view.entities.summary("Relationships").get_text_of("Host")
+        host, = [host for host in provider.hosts.all() if host.name == host_name]
+        host_data, = [data for data in provider.data['hosts'] if data['name'] == host.name]
+        host.update_credentials_rest(credentials=host_data['credentials'])
```

Comment:
There are two options to do this operation means updating host credentials. Either you can go with a separate fixture or add a simple method to do it for you.